ethereum/spec/src/test/java/tech/pegasys/teku/spec/logic/StateUpgradeTransitionTest.java

```diff
@@ -93,6 +93,11 @@ public void setup(final SpecContext specContext) {
             afterBeaconStateClass = BeaconStateElectra.class;
             yield TestSpecFactory.createMinimalWithElectraForkEpoch(milestoneTransitionEpoch);
           }
+          case FULU -> {
```

Comment:
do we need to add FULU to  `TestSpecContext` also?

Reply:
Ooopsie :)